java/array/FizzBuzz.java

```diff
@@ -29,12 +29,36 @@ Given an integer n, return a string array answer (1-indexed) where:
 
 public class FizzBuzz {
     public static void main(String[] args) {
-        System.out.println(fizzBuzz(1));
+        System.out.println(fizzBuzz(15));
     }
 
     public static List<String> fizzBuzz(int n) {
         //TODO: implement this method
         List<String> result = new ArrayList<>();
+        for(int i=1;i<=n;i++)
+        {
+
+        if(i%3==0 && i%5==0)
+        {
+            
+            result.add("FizzBuzz");
+            continue;
+        }
+        else if(i % 3 == 0)
+        {
+            
+            result.add("Fizz");
+            continue;
+        }
+        else if(i % 5 == 0)
+        {
+            result.add("Buzz");
+            continue;
+        }
+        else{
+        result.add(Integer.toString(i));
+        }     
+        }
```

Comment:
Please format the code, the logic looks good.

Reply:
okk..

Reply:
Hey, I have formatted a code please check it...